src/routes/transactions/transactions.controller.ts

```diff
@@ -233,6 +233,8 @@ export class TransactionsController {
     timezoneOffsetMs: number,
     @Query('trusted', new DefaultValuePipe(true), ParseBoolPipe)
     trusted: boolean,
+    @Query('imitation', new DefaultValuePipe(false), ParseBoolPipe)
```

Comment:
I'm not against this, just curious: we are filtering out imitation transfers by default, right? Is that the clients expected behaviour?

Reply:
This was based on the demo implementation. As it directly affects the clients, setting it to `true` by default makes the most sense. I changed it in c2473aa1c93d665a2ed412471be74bd38e08c9ed.